slick2d-examples/nb-configuration.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not too keen on putting IDE-specific file in the repository (Be it Netbeans or Eclipse or something else). I think if we were to try to enforce code formatting rules I'd prefer to do it with Maven (via Checkstyle for instance), or with an [.editorconfig](http://editorconfig.org/) file